spec/acceptance/server/recovery_spec.rb

```diff
@@ -59,26 +48,7 @@ class { 'postgresql::globals':
         class { 'postgresql::server': }
       EOS
 
-      apply_manifest(pp, catch_failures: true)
-      apply_manifest(pp, catch_changes: true)
-    end
-
-    describe file('/tmp/recovery.conf') do
-      it { is_expected.not_to be_file }
-    end
-  end
-
-  describe 'should not manage recovery' do
```

Comment:
is this tested elsewhere ?

Reply:
Was fifty/fifty on removing it. Did not understand what it was actually testing as the only check was if the file existed, and given as it wasn't mentioned in the manifest it never would, something covered in the above test.

Reply:
There also is a unit test that test's setting that value to false.